code-challenges.js

```diff
@@ -9,60 +9,186 @@
 // --------------------1) Create a function that determines if a temperature is below boiling point, at boiling point or above boiling point. Boiling point is 212 degrees Farhenheit.
 // Use the test variables provided below. Expected outcome: "35 is below boiling point", "350 is above boiling point", "212 is at boiling point"
 
-var temp1 = 35
-var temp2 = 350
-var temp3 = 212
+// var temp1 = 35
+// var temp2 = 350
+// var temp3 = 212
 
+// const boilingPoint = (num) => {
+//     if(num > 212){
+//         return `${num} is above boiling point`
+//     } else if (num < 212){
+//         return `${num} is below boiling point`
+//     } else if (num == 212) {
+//         return `${num} is at boiling point`
+//     } else {
+//         return "something went wrong"
+//     }
+// }
+
+// console.log(boilingPoint(temp1));
+// console.log(boilingPoint(temp2));
+// console.log(boilingPoint(temp3));
 
 
 
 
 // --------------------2) Create a function that multiplies each number in the array by 5 using a for loop.
 // Use the test variable provided below. Expected outcome: [15, 35, 0, 30, -45]
 
-var myNumbers1 = [3, 7, 0, 6, -9]
-
+// var myNumbers1 = [3, 7, 0, 6, -9]
+// var newNumbers = []
 
+// const multiplier = (array) => {
+//     for (let index = 0; index < array.length; index++) {
+//         newNumbers.push(array[index] * 5)
+//     }
+//     return newNumbers
+// }
```

Comment:
this looks great! I just would move that variable with the empty array into the function, above the loop to keep it all contained :)